core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -4390,6 +4401,2215 @@ class KafkaApisTest extends Logging {
     assertEquals("broker2", node.host)
   }
 
+  private def expectedAcquiredRecords(firstOffset : Long, lastOffset : Long, deliveryCount : Int) : util.List[AcquiredRecords] = {
```

Comment:
nit: would it better to have these methods defines later when used in tests?

Reply:
Hi thanks for reviewing. I think the first occurrence of these methods is in the test just underneath them. Isn't that what you mean here ?

Reply:
it's not that important but generally the helper method will come post the usage i.e. test -> helper method.